### ISSUE TITLE
Accessibility rules states that links to PDF documents should have visible references

### ISSUE THREAD

**Issue by NeilWJames** [wp-document-revisions/wp-document-revisions#322]
Use of the Accessibilty Checker states that guidelines indicate links to PDF documents should have a visible reference that it is to a PDF document.

Many documents are PDF documents. The WPDR blocks (and shortcodes/widgets) contain lists of documents.

These should have a Display option `Show PDF file indication` (default false) which when set to true will add ` <small>(PDF)</small>` to the link for PDF documents.

Also the Latest Documents block (and Recently Revised Documents widget) will output the title with a heading level 2 and the documents with heading level 3. If there is no list title the heading level 2 is not output. This missing header level does not follow guidelines.  So in this case the doceuments should be output using heading level 2.
